### ISSUE TITLE
Many mounts are missing the racial requirements.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#214]
Placeholder.
Taurens + kodos
Battlestriders + gnomes/dorfs
PVP mounts exceptions.

**Comment by Cabro** [comment]
Yes, other races CAN ride kodos, but taurens should ONLY ride kodos.
Same as battlestriders only ridable by gnomes and dwarves, not humans and night elves.

**Comment by GuybrushGit** [comment]
Taurens can ride wolves.

**Comment by Cabro** [comment]
https://wow.gamepedia.com/Patch_3.0.8
https://wow.gamepedia.com/Mechanostrider - here read the part about the gnomes and dwarves only at Patch 3.0.8 line

I did some more digging and indeed you are right, taurens were able to mount both kodos and wolves from the horde faction mounts.


Note that the PvP mounts had some special racial restrictions. Still digging on that.

**Comment by GuybrushGit** [comment]
Available to all:
http://web.archive.org/web/20060719123714/http://www.worldofwarcraft.com:80/info/basics/mounts/wolves.html
http://web.archive.org/web/20060719021256/http://www.worldofwarcraft.com:80/info/basics/mounts/horses.html
http://web.archive.org/web/20060720014605/http://www.worldofwarcraft.com:80/info/basics/mounts/nightsabers.html
http://web.archive.org/web/20060720120907/http://www.worldofwarcraft.com:80/info/basics/mounts/rams.html
http://web.archive.org/web/20060719123447/http://www.worldofwarcraft.com:80/info/basics/mounts/kodos.html

Gnome and Dwarves:
http://web.archive.org/web/20060720230225/http://www.worldofwarcraft.com:80/info/basics/mounts/mechanostriders.html

Troll, Orc, Undead:
http://web.archive.org/web/20060719123858/http://www.worldofwarcraft.com:80/info/basics/mounts/raptors.html
http://web.archive.org/web/20060719125806/http://www.worldofwarcraft.com:80/info/basics/mounts/skeletalhorses.html
